src/PureCloud.Client/Apis/IntegrationsApi.cs

```diff
@@ -2097,7 +2079,7 @@ public async Task<IntegrationConfiguration> UpdateIntegrationConfigCurrentAsync(
 
         var client = _httpClientFactory.CreateClient(PureCloudConstants.PureCloudClientName);
 
-        var uri = UriHelper.GetUri($"api/v2/integrations/{integrationId}/config/current", parameters);
+        var uri = UriHelper.GetUri($"api/v2/integrations/{Uri.EscapeDataString(integrationId)}/config/current", parameters);
```

Comment:
Why is parameters collection is needed? If it is not used, don't initialize it and then no need to use GetUri at all. Apply the same feedback everywhere